### PR TITLE
fixing typo in device access error

### PR DESCRIPTION
**Pull request by rajasec**
Fixed the typo in error for device access.
Signed-off-by: rajasec <rajasec79@gmail.com>

### CODE REVIEW

File: spec.go [hunk 1]
```diff
@@ -385,7 +385,7 @@ func createCgroupConfig(name string, spec *specs.Spec) (*configs.Cgroup, error)
 			minor = *d.Minor
 		}
 		if d.Access == nil || *d.Access == "" {
-			return nil, fmt.Errorf("device access at %d field canot be empty", i)
+			return nil, fmt.Errorf("device access at %d field can not be empty", i)
```
Comment (mikebrow) [inline spec.go]: For this sentence, I believe the proper spelling + grammar is 'cannot.'  :-)

**Comment by rajasec** [comment]
My initial commit was cannot :) , then later I changed to can not

**Comment by mikebrow** [comment]
While I can not say that the second one is better I can say that the first one was :-)

**Comment by rajasec** [comment]
I'm repushing my first commit, because this can interpreted in different way.
Thanks @mikebrow.

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**